src/drivers/build/install-nvidia-drivers

```diff
@@ -33,7 +33,8 @@ function nvidiaPresent {
 }
 
 echo ======== If NVIDIA present exit early =========
-if [ nvidiaPresent ] ; then
+nvidiaPresent
```

Comment:
What's the difference

Reply:
previous way bash will always take true path, I think bash treat it as no null value. later way bash will check its return value.